.github/workflows/macos.yml

```diff
@@ -78,7 +76,7 @@ jobs:
 
       - name: Configure Build
         env:
-          folly_SOURCE: BUNDLED #brew folly does not have int128
+          fmt_SOURCE: BUNDLED #picked up brew fmt (from ccache) is too new
```

Comment:
Folly doesn't need to be force bundled because it is not coming from brew anymore but is installed by the setup script.
It uses fmt as bundled because the fmt version is too new. This need to be undone once the build script picks up the installed version via the setup script (and not the one present in homebrew from some dependency).